specification/orbital/resource-manager/Microsoft.Orbital/aodg/preview/2024-03-01-preview/aodg.json

```diff
@@ -0,0 +1,1750 @@
+        }
+      },
+      "description": "One of the partner routers within a global communications site."
+    },
+    "LocationData": {
+      "type": "object",
+      "required": [
+        "city",
+        "district",
+        "country"
+      ],
+      "properties": {
+        "city": {
+          "type": "string",
+          "description": "The city of the location (e.g., Miami)."
+        },
+        "district": {
+          "type": "string",
+          "description": "For example, the state (e.g., Florida), province (e.g., Ontario), or region (e.g., Tuscany) of the location."
+        },
+        "country": {
+          "type": "string",
+          "description": "The nation (e.g., United States of America) of the location."
+        },
+        "closestAzureRegion": {
+          "type": "string",
+          "description": "The name of the closest Azure region, (e.g., westus2)."
+        },
+        "extra": {
+          "type": "string",
+          "x-nullable": true,
+          "description": "Any extra information needed to uniquely identify the location (e.g., zip code)."
+        }
+      },
+      "description": "Describes a location outside of Azure."
+    },
+    "GlobalCommunicationsSiteType": {
+      "type": "string",
+      "enum": [
+        "Edge",
+        "GroundStation"
+      ],
+      "x-ms-enum": {
+        "name": "GlobalCommunicationsSiteType",
+        "modelAsString": true
+      },
+      "description": "The type of the global communications site. groundStations can only use sites of type GroundStation, and edgeSites can only use sites of type Edge."
+    },
+    "GlobalCommunicationsSitesProperties": {
+      "type": "object",
+      "properties": {
+        "locationData": {
+          "allOf": [
+            {
+              "$ref": "#/definitions/LocationData"
+            }
+          ],
+          "description": "Describes the location of the Azure Orbital Digital Ground site."
+        },
+        "globalCommunicationsSiteType": {
+          "type": "string",
+          "allOf": [
+            {
+              "$ref": "#/definitions/GlobalCommunicationsSiteType"
+            }
+          ],
+          "description": "The type of the Azure Orbital Digital Ground site. EdgeSite or GroundStation."
+        },
+        "partnerRouters": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/PartnerRouter"
+          },
+          "x-ms-identifiers": [],
+          "description": "Partner routers connected to this site."
+        },
+        "partnerSiteBandwidth": {
+          "type": "integer",
+          "format": "int64",
+          "minimum": 0,
+          "readOnly": true,
+          "description": "The bandwidth per partner router cross-connection at this Azure Orbital Digital Ground site, measured in bits per second."
+        }
+      },
+      "description": "Properties object for a Azure Orbital Digital Ground site."
```

Comment:
The description doesn't match with the property name, and same is true for various other descriptions. Please correct these.

Reply:
Updated